src/main/java/com/alkemy/ong/service/impl/EmailServiceImpl.java

```diff
@@ -0,0 +1,85 @@
+package com.alkemy.ong.service.impl;
+
+import com.alkemy.ong.exception.EmailException;
+import com.alkemy.ong.service.IEmailService;
+
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.Content;
+import com.sendgrid.helpers.mail.objects.Email;
+import com.sendgrid.helpers.mail.objects.Personalization;
+
+import org.springframework.stereotype.Service;
+import org.springframework.beans.factory.annotation.Value;
+
+import org.springframework.http.HttpStatus;
+
+@Service
+public class EmailServiceImpl implements IEmailService {
+
+    @Value("${app.sendgrid.sender}")
+    private String sender;
+
+    @Value("${app.sendgrid.apiKey}")
+    private String apiKey;
+
+    @Value("${app.sendgrid.templateIdRegister}")
+    private String templateIdRegister;
+
+    public void send(Mail mail) throws Exception {
+
+        SendGrid sendGrid = new SendGrid(apiKey);
+        Request request = new Request();
+
+        try {
+            request.setMethod(Method.POST);
+            request.setEndpoint("mail/send");
+            request.setBody(mail.build());
+            Response response = sendGrid.api(request);
+
+            if (response.getStatusCode() != HttpStatus.ACCEPTED.value()) {
+                throw new EmailException();
+            }
+
+        } catch (EmailException ex) {
+            System.out.println("Error sending email");
+        }
+    }
+
+    @Override
+    public void sendRegisterConfirmation(String receiverEmail, String name) throws Exception {
+
+        Email from = new Email(sender);
+        Email to = new Email(receiverEmail);
+
+        Personalization personalization = new Personalization();
+        personalization.addTo(to);
+        personalization.setSubject("Welcome to SomosMas ONG!");
```

Comment:
Esto podria hacerde de varias maneras, se podria tener una variable de clase para poner el texto alli y pasarle a este metodo directamente esa variable:
`private static final WELCOME_TEXT = "Welcome to SomosMas ONG!";`
o incluso mejor, se podria crear una clase utils que tenga las constantes que vayamos a utilizar en todo lo relativo al envio de mail, porque van a ir apareciendo un par mas